NEWS.rst

```diff
@@ -16,6 +16,10 @@ also been tested on PyPy2.7.13 v7.1.1 and PyPy3.6.1 v7.1.1-beta0.
 
 Note we intend to drop Python 2.7 support in early 2020.
 
+``Bio.motifs.PositionSpecificScoringMatrix.search()`` method has been
+re-written, it is now very fast, but doesn't require a lot of memory for long
+sequences, unlike ``.calculate()``.
```

Comment:
The comparison to `calculate` is not quite correct. The purpose of `calculate` is to calculate the motif score for every position along the sequence. For long sequences, the value returned by `calculate` needs a lot of memory, but this is what the user asked for. Internally, the `calculate` function requires very little memory.

Reply:
I'm happy to change that, but I'm not sure what's wrong with what I wrote for the end-user experience: `calculate` requires a lot of memory, `search` doesn't.

Reply:
How is the new description?

Reply:
It's fine now, thanks!